src/visit_vtk/lightweight/vtkElementLabelArray.h

```diff
@@ -39,8 +39,8 @@ class VISIT_VTK_LIGHT_API vtkElementLabelArray : public vtkCharArray
   virtual ~vtkElementLabelArray();
   std::map<std::string, LabelToZoneMapping> NameMapping;
 private:
-  vtkElementLabelArray(const vtkElementLabelArray&);   // not implemented
-  void operator=(const vtkElementLabelArray&);  // not implemented
+  vtkElementLabelArray(const vtkElementLabelArray&) = delete;
+  void operator=(const vtkElementLabelArray&) = delete;
```

Comment:
again. isn't this different? doesn't the compiler generate a default operator=? Do we not want it to?